src/ukify/ukify.py

```diff
@@ -706,6 +730,24 @@ def make_uki(opts):
     print(f"Wrote {'signed' if sign_args_present else 'unsigned'} {opts.output}")
 
 
+def pe_read_sections(uki: str):
+    pe = pefile.PE(uki, fast_load=True)
+    print(f"Sections in {uki}:")
+    for section in pe.sections:
+        name = section.Name.decode().rstrip('\x00').encode('utf-8')
+        if not name in Section.sections_to_show:
```

Comment:
Why restrict the sections listed? We allow adding arbitrary sections, so we should also allow reading them.

Reply:
There's a lot of sections that are part of systemd-stub, since we are using that as base to build the UKI (.text, and so on). That's why I wanted to filter out only the UKI sections that interest us, and not print a normal program sections

Reply:
I think this is a non-issue. But it's better to either filter them out or provide a `--verbose` mode or so to list all sections.

Reply:
ok, but the "unknown" sections unless specified via `--section` will be treated as `binary`